### PR TITLE
Added getting started for Adding GTK# Backend to Xamarin.Forms project

### PR DESCRIPTION
**Pull request by duanenewman**
based on my article: http://duanenewman.net/blog/post/using-prism-with-xamarin-forms-and-gtk/

### CODE REVIEW

**Comment by dansiegel** [comment]
Following the decision to drop UWP as implemented, @brianlagunas and I have decided that we'll actually be moving all of the navigation pieces back to Prism Forms which will actually simplify the issue with trying to get access to the Navigation interfaces in GTK#... that said I do think that this could still be great documentation in general for those that may be looking at adding an alternate platform.

**Comment by duanenewman** [comment]
👌 So this won't specifically be an issue moving forward.. do you want this, or a similar article to live anywhere in the docs? Also, what is the roadmap for moving the the navigation back to XF? If it is far enough out for you guys I may take a crack at a PR for those changes.

**Comment by dansiegel** [comment]
Moving the Navigation interfaces back to Prism.Forms will be done as part of the IInitialize changes....

File: docs/xamarin-forms/gtk/Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<Reference Include="Prism, Version=7.1.0.431, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59, processorArchitecture=MSIL">
+    <HintPath>..\packages\Prism.Core.7.1.0.431\lib\net45\Prism.dll</HintPath>
+</Reference>
+<Reference Include="Prism.Forms, Version=7.1.0.431, Culture=neutral, processorArchitecture=MSIL">
+    <HintPath>..\packages\Prism.Forms.7.1.0.431\lib\net45\Prism.Forms.dll</HintPath>
+</Reference>
+```
+
+To NetStandard 2.0 references like these (`..\packages\Prism.Core.7.1.0.431\lib\netstandard2.0\*.dll`):
+
```
Comment (dansiegel) [inline docs/xamarin-forms/gtk/Getting-Started.md]: This whole section can be removed. If you can update this, the rest of it looks good to me. Prism 7.2 shouldn't have this issue

**Review by dansiegel — CHANGES_REQUESTED** [review]

File: docs/xamarin-forms/gtk/Getting-Started.md (hunk 1 above)
Comment (duanenewman) [reply docs/xamarin-forms/gtk/Getting-Started.md]: I removed the content related to forcing the netstandard reference.

**Review by dansiegel — APPROVED** [review]

**Pull request merged**